.travis.yml

```diff
@@ -28,17 +28,17 @@ addons:
       - llvm-toolchain-precise
 
 env:
-  - COMPILER=clang++-3.6 CMAKE_GENERATOR="Ninja"
-  - COMPILER=clang++-3.6 CMAKE_GENERATOR="Unix Makefiles"
-
-  - COMPILER=clang++-3.7 CMAKE_GENERATOR="Unix Makefiles"
-  - COMPILER=clang++-3.7 CMAKE_GENERATOR="Ninja"
-
-  - COMPILER=clang++-3.8 CMAKE_GENERATOR="Unix Makefiles"
-  - COMPILER=clang++-3.8 CMAKE_GENERATOR="Ninja"
-
-  - COMPILER=gcc CMAKE_GENERATOR="Ninja"
-  - COMPILER=gcc CMAKE_GENERATOR="Unix Makefiles"
+  # Test all generators and Clang + GCC
+  - TEST=true COMPILER=clang++-3.8 CMAKE_GENERATOR="Ninja"
+  - TEST=true COMPILER=clang++-3.8 CMAKE_GENERATOR="Unix Makefiles"
+  - TEST=true COMPILER=gcc         CMAKE_GENERATOR="Ninja"
+  - TEST=true COMPILER=gcc         CMAKE_GENERATOR="Unix Makefiles"
+
+  # Benchmarks
+  - BENCHMARK=true COMPILER=clang++-3.6 CMAKE_GENERATOR="Unix Makefiles"
+  - BENCHMARK=true COMPILER=clang++-3.7 CMAKE_GENERATOR="Unix Makefiles"
+  - BENCHMARK=true COMPILER=clang++-3.8 CMAKE_GENERATOR="Unix Makefiles"
+  - BENCHMARK=true COMPILER=gcc         CMAKE_GENERATOR="Unix Makefiles"
```

Comment:
> I can add testing on all Clang versions if you think it is useful

It would be useful to test a few versions so we can check whether the detection of supported standards works.

>  I would definitely benchmark on many versions.

Makes sense of course, but I see a couple of challenges:

1. How do we push the results produced by the different Travis jobs to gh_pages?
2. Metabench would then become a tool for comparing compilers besides the original intent of comparing different implementations of metaprogramming techniques. Do we want that?